.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to [Drash Middleware](https://github.com/drashland/deno-drash-middleware/)
+
+## Bug Reports
+
+A bug is a *demonstrable problem* that is caused by the code in the repository. Good bug reports are extremely helpful, so thanks!
+If you want to report a bug, click [here](https://github.com/drashland/deno-drash-middleware/issues/new?assignees=&labels=bug&template=bug_report.md&title=).
+
+## Feature Requests
+Feature requests are welcome. But take a moment to find out whether your idea fits with the scope and aims of the project. It's up to *you* to make a strong case to convince the project developer of the merits of this feature. Please provide as much detail and context as possible. If you want to request a feature, click [here](https://github.com/drashland/deno-drash-middleware/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=).
+
+## Pull Requests
+
+Please **ask first** before embarking on any significant pull request (e.g. implementing features, refactoring code), otherwise you risk spending a lot of time working on something that the project's developers might not want to merge into the project.
+
+1. [Fork](https://help.github.com/articles/fork-a-repo/) the project, clone your fork, and configure the remotes:
+    ```bash
+    # Clone your fork of the repo into the current directory
+    git clone https://github.com/<your-username>/deno-drash-middleware.git
+    # Navigate to the newly cloned directory
+    cd deno-drash
+    # Assign the original repo to a remote called "upstream"
+    git remote add upstream https://github.com/drashland/deno-drash-middleware.git
+    ```
+2. If you cloned a while ago, get the latest changes from upstream:
+    ```bash
+    git checkout master
+    git pull upstream master
+    ```
+3. Create a new topic branch (off the main project development branch) to contain your feature, change, or fix:
+    ```bash
+    git checkout -b <topic-branch-name>
+    ```
+4. Create the directories and files to hold your middleware. Your middleware should be exported from it's own `mod.ts`:
+    ```bash
+    mkdir tests/<middleware>
+    mkdir src/<middleware>
+    touch src/<middleware>/mod.ts
+    touch src/<middleware>/related_files.ts
```

Comment:
touch src/<middleware>/README.md

Reply:
Added. see the closest commit below. I also removed the `src` bit after seeing your comments on the #1 PR